docs/_basic/ja_listening_events.md

```diff
@@ -37,13 +37,16 @@ app.event('team_join', async ({ event, client }) => {
 <div class="secondary-content" markdown="0">
 `message()` リスナーは `event('message')` と等価の機能を提供します。
 
-イベントのサブタイプをフィルタリングしたい場合、組み込みの `subtype()` ミドルウェアを使用できます。 `bot_message` や `message_replied` のような一般的なメッセージサブタイプの情報は、[メッセージイベントのドキュメント](https://api.slack.com/events/message#message_subtypes)を参照してください。
+イベントのサブタイプをフィルタリングしたい場合、組み込みの `subtype()` ミドルウェアを使用できます。 `message_changed` や `message_replied` のような一般的なメッセージサブタイプの情報は、[メッセージイベントのドキュメント](https://api.slack.com/events/message#message_subtypes)を参照してください。
 </div>
 
 ```javascript
-// bot からのメッセージ全てと一致
-app.message(subtype('bot_message'), ({ message }) => {
-  console.log(`The bot user ${message.user} said ${message.text}`);
+// パッケージから subtype をインポート
+const { App, subtype } = require('@slack/bolt');
+
+// user からのメッセージの編集と一致
+app.message(subtype('message_changed'), ({ event }) => {
+  console.log(`The user ${event.message.user} changed their message from ${event.previous_message.text} to ${event.message.text}`);
```

Comment:
I know the usage of console is not your change but we may want to improve the part as well. Can you change the code this way? It's the same for the English one.

```suggestion
app.message(subtype('message_changed'), ({ event, logger }) => {
  logger.info(`The user ${event.message.user} changed their message from ${event.previous_message.text} to ${event.message.text}`);
```

Reply:
Yeah, that makes sense.  Maybe we should change it for the entirety of the docs? It looks like `console.log` is used throughout at the moment.

Reply:
Oh I see. In the case, let's work on the replacement in a different issue/PR!